boot.php

```diff
@@ -2271,21 +2243,24 @@ function is_site_admin() {
  *
  * @return string
  */
-function build_querystring($params, $name=null) {
+function build_querystring($params, $name = null) {
 	$ret = "";
-	foreach ($params as $key=>$val) {
+	foreach ($params as $key => $val) {
 		if (is_array($val)) {
-			if ($name==null) {
+			/// @TODO maybe not compare against null, use is_null()
```

Comment:
`is_null($name)` is exactly the same as `$name === null` in this context, but here it's only `==`, which may be an oversight from the original developer.